src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -27,6 +27,7 @@ public class JsonPackageSpecReader
         public static readonly string PackOptions = "packOptions";
         public static readonly string PackageType = "packageType";
         public static readonly string Files = "files";
+        public static readonly FileShare share = FileShare.ReadWrite | FileShare.Delete;
```

Comment:
yeah i just read the constructor description

---
In reply to: [121762845](https://github.com/NuGet/NuGet.Client/pull/1487#discussion_r121762845) [](ancestors = 121762845)